internal/providers/terraform/azure/app_service_plan.go

```diff
@@ -0,0 +1,114 @@
+package azure
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/infracost/infracost/internal/schema"
+	log "github.com/sirupsen/logrus"
+
+	"github.com/shopspring/decimal"
+)
+
+func GetAzureRMAppServicePlanRegistryItem() *schema.RegistryItem {
+	return &schema.RegistryItem{
+		Name:  "azurerm_app_service_plan",
+		RFunc: NewAzureRMAppServicePlan,
+	}
+}
+
+func NewAzureRMAppServicePlan(d *schema.ResourceData, u *schema.UsageData) *schema.Resource {
+	sku := d.Get("sku.0.size").String()
+	skuRefactor := ""
+	os := "Windows"
+	capacity := d.Get("sku.0.capacity").Int()
+	location := d.Get("location").String()
+	productName := "Standard Plan"
+
+	switch sku[2:] {
+	case "v2":
+		skuRefactor = sku[:2] + " " + sku[2:]
+		productName = "Premium v2 Plan"
+	case "v3":
+		skuRefactor = sku[:2] + " " + sku[2:]
+		productName = "Premium v3 Plan"
+	}
+
+	switch sku {
```

Comment:
Can this code be made a little more future-proof so when Azure add PC5 or S4 or B4, it continues to work? For example, if the first two letters are PC, then use that + the 3rd letter for the skuRefactor. If it starts with "B" then do something similar, same when it starts with "S"